reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/add-infra-node.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
Completely agree. This is just a stop gap until ASG. When cleaning up both scenarios kind of stink in regardless of pure ansible and a new stack because there is really no workflow in destroying the environment. I'm definitely not looking forward to going back to potential API errors when launching with pure ansible. Ive been spoiled by the 100% consistency when launching with cloudformations. But going back this is just a stop gap of the ASG work that I really need to visit.